ranger-zookeeper/src/test/java/com/flipkart/ranger/zookeeper/model/ServiceNoProviderTest.java

```diff
@@ -53,58 +51,16 @@ public void stopTestCluster() throws Exception {
         }
     }
 
-    private static final class TestShardInfo {
-        private int shardId;
-
-        public TestShardInfo(int shardId) {
-            this.shardId = shardId;
-        }
-
-        public TestShardInfo() {
-        }
-
-        public int getShardId() {
-            return shardId;
-        }
-
-        public void setShardId(int shardId) {
-            this.shardId = shardId;
-        }
-
-        @Override
-        public boolean equals(Object o) {
-            if (this == o) {
-                return true;
-            }
-            if (o == null || getClass() != o.getClass()) {
-                return false;
-            }
-
-            TestShardInfo that = (TestShardInfo) o;
-
-            if (shardId != that.shardId) {
-                return false;
-            }
-
-            return true;
-        }
-
-        @Override
-        public int hashCode() {
-            return shardId;
-        }
-    }
-
     @Test
-    public void testBasicDiscovery() throws Exception {
-        SimpleShardedServiceFinder<TestShardInfo> serviceFinder = ServiceFinderBuilders.<TestShardInfo>shardedFinderBuilder()
+    public void testBasicDiscovery() {
+        SimpleShardedServiceFinder<TestNodeData> serviceFinder = ServiceFinderBuilders.<TestNodeData>shardedFinderBuilder()
```

Comment:
User `var` to create mutalble variables like `val` counterpart

Reply:
The serviceRegistry was always a List<>, the nodeList getting updated via updateNodes after refresh was always a list Do you want me to change that to a set? 

We earlier wanted to change ServiceDataSource to a set.